### PR TITLE
Command driver support

### PR DESCRIPTION
**Pull request by simongdavies**
# What does this change
Adds support to Porter for Command Driver.
Updates cnab-go dependency to v0.7.0-beta1

### CODE REVIEW

File: pkg/porter/cnab.go [hunk 1]
```diff
@@ -304,9 +306,14 @@ func (o *sharedOptions) defaultDriver() {
 // validateDriver validates that the provided driver is supported by Porter
 func (o *sharedOptions) validateDriver() error {
 	switch o.Driver {
-	case "docker", "debug":
+	case "docker", "debug", "kubernetes":
```
Comment (carolynvs) [inline pkg/porter/cnab.go]: I'm not sure why kubernetes was added in this PR? We haven't done any testing so I'm not sure if there's more work to be done on our side to make sure that we are properly loading and configuring that driver. Let's split that change into a separate PR and add an integration test for it.

Comment (simongdavies) [reply pkg/porter/cnab.go]: OK, I removed the Kubernetes driver.
